Dockerfile

```diff
@@ -1,26 +1,21 @@
-FROM rhel7
+# Using playbook2image as a base
+# See https://github.com/aweiteka/playbook2image for details on the image
+# including some of the settings below
+FROM docker.io/aweiteka/playbook2image:latest
 
-MAINTAINER Troy Dawson <tdawson@redhat.com>
+# The playbook to be run is specified via the PLAYBOOK_FILE env var
+# Setting a default of openshift_facts.yml as it's an informative playbook
+# that can help test that everything is set properly (inventory, sshkeys)
+ENV PLAYBOOK_FILE=plabyooks/byo/openshift_facts.yml \
+    ANSIBLE_HOST_KEY_CHECKING=False \
```

Comment:
Right... for the use cases of running the built image from inside OpenShift itself I was thinking to use a ConfigMap for these settings. For a standalone container I'll update the docs to explain that better.

One thing to keep in mind with mounting `~/.ssh` is that perms/selinux come into play. That's why the example `docker run` in `README_CONTAINER_IMAGE` sets the uid with `id -u` and mounts the ssh key with `:Z` - this has a side effect or leaving your key labelled something like `system_u:object_r:container_file_t:s0:c609,c894`